Hybrid Key trust Prerequisites: add prerequisites checklist - PR for Issue #3248 (#3616)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md

Key trust deployments do not need client issued certificates for on-premises authentication.  Active Directory user accounts are automatically configured for public key mapping by Azure AD Connect synchronizing the public key of the registered Windows Hello for Business credential to an attribute on the user's Active Directory object.

The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third-party enterprise certification authority. The detailed requirements for the Domain Controller certificate are shown below.

* The certificate must have a Certificate Revocation List (CRL) distribution point extension that points to a valid CRL.
* Optionally, the certificate Subject section should contain the directory path of the server object (the distinguished name).
* The certificate Key Usage section must contain Digital Signature and Key Encipherment.

> * Optionally, the certificate Basic Constraints section should contain:

Reviewer: ```suggestion
* Optionally, the certificate Basic Constraints section should contain [Subject Type=End Entity, Path Length Constraint=None].
```

Key trust deployments do not need client issued certificates for on-premises authentication.  Active Directory user accounts are automatically configured for public key mapping by Azure AD Connect synchronizing the public key of the registered Windows Hello for Business credential to an attribute on the user's Active Directory object.

> The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can use a 3rd Party enterprise certification authority too. The detailed requieriments for the Domain Controller certificate are shown below.

Reviewer 2: ```suggestion
The minimum required enterprise certification authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third party enterprise certification authority. Detailed requirements for Domain Controller certification are as follows:
```

> The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can use a 3rd Party enterprise certification authority too. The detailed requieriments for the Domain Controller certificate are shown below.

Reviewer: ```suggestion
The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third-party enterprise certification authority.. The detailed requirements for the Domain Controller certificate are shown below.
```

Reviewer 2: ```suggestion
The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third-party enterprise certification authority. The detailed requirements for the Domain Controller certificate are shown below.

```

> The minimum required enterprise certificate authority that can be used with Windows Hello for Business is Windows Server 2012, but you can use a 3rd Party enterprise certification authority too. The detailed requieriments for the Domain Controller certificate are shown below.

Reviewer 2: ```suggestion
The minimum required enterprise certification authority that can be used with Windows Hello for Business is Windows Server 2012, but you can also use a third-party enterprise certification authority. The detailed requirements for the Domain Controller certificate are shown below.
```

> * The certificate must have a CRL distribution-point extension that points to a valid certificate revocation list (CRL).

Reviewer: ```suggestion
* The certificate must have a Certificate Revocation List (CRL) distribution-point extension that points to a valid CRL.
```

> * Optionally, the certificate Subject section should contain the directory path of the server object (the distinguished name) 

Reviewer: ```suggestion
* Optionally, the certificate Subject section should contain the directory path of the server object (the distinguished name).
```

> * The certificate Key Usage section must contain:

Reviewer: ```suggestion
* The certificate Key Usage section must contain Digital Signature and Key Encipherment.
```.

> * The certificate Key Usage section must contain:

Reviewer: ```suggestion
* The certificate Key Usage section must contain Digital Signature and Key Encipherment.
```

Digital Signature, Key Encipherment

> * Optionally, the certificate Basic Constraints section should contain:

Reviewer: ```suggestion
* Optionally, the certificate Basic Constraints section should contain [Subject Type=End Entity, Path Length Constraint=None].
```

> [Subject Type=End Entity, Path Length Constraint=None]

Reviewer: ```suggestion
```

> * The certificate Enhanced Key Usage section must contain: Client Authentication (1.3.6.1.5.5.7.3.2) and Server Authentication (1.3.6.1.5.5.7.3.1)

Reviewer: ```suggestion
* The certificate Enhanced Key Usage section must contain Client Authentication (1.3.6.1.5.5.7.3.2) and Server Authentication (1.3.6.1.5.5.7.3.1).
```

* The certificate Subject Alternative Name section must contain the Domain Name System (DNS) name.  

> * The certificate template must have an extension that has the BMP data value "DomainController."

Reviewer: ```suggestion
* The certificate template must have an extension that has the BMP data value "DomainController".
```

> * The domain controller certificate must be installed in the local computer's certificate store

Reviewer: ```suggestion
* The domain controller certificate must be installed in the local computer's certificate store.
```
